pkgs/servers/varnish/digest.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchpatch, fetchFromGitHub, autoreconfHook, pkgconfig, varnish, libmhash, docutils }:
```

Comment:
just a result of copy-paste from ```geoip.nix``` nearby :(
BTW, there was nix-lint project, abandoned long time ago.
Any plans to revival? It could save a lot ot reviewers time and make the codebase cleaner (there must be hundreds of unused params, missign spaces, tab-characters, .. in nixpkgs).